documentation/testnet/getting_started/04_transfer_credits.md

```diff
@@ -0,0 +1,49 @@
+---
+id: transfer_credits
+title: Public and Private Transfers
+sidebar_label: Transfer Credits
+---
+
+The `credits.aleo` program allows for the transfer of credits in a public or private manner, along with conversions
+between the two forms.
+
+## Transfer Functions
+
+### `transfer_private`
+
+The `transfer_private` function enables a private transfer of credits between two addresses.
+
+### `transfer_public`
+
+The `transfer_public` function is used to perform a public transfer of credits between two addresses.
+
+### `transfer_private_to_public`
+
+The `transfer_private_to_public` function is used to convert a private credit into a public credit.
+### `transfer_public_to_private`
+
+The `transfer_public_to_private` function is used to convert a public credit into a private credit.
+
+## Function Usage
+
+To use these functions, one needs to craft a `credits.aleo` program execution. Here is an example:
+
+```
+snarkos developer transfer <AMOUNT_TO_TRANSFER> --input-record <INPUT_RECORD> --recipient <RECIPIENT_ADDRESS> --private-key <PRIVATE_KEY> --query "<http://localhost:3030>" --broadcast "<http://localhost:3030/testnet3/transaction/broadcast>"
```

Comment:
Can you confirm wether or not this command takes `--input-record` or `--input_record`
same for the private key.

setting the endpoints to http://localhost:3030 assumes that the user has already started a snarkos node in addition to installing the CLI. Can you clarify that those steps are required or link to the documentation that you copied this line from?

Reply:
@collinc97 Good catch! Will do.

Reply:
Just confirmed the correct usage of the command and will update docs to match. 

![Screen Shot 2023-06-26 at 12 23 14 PM](https://github.com/AleoHQ/welcome/assets/37510951/b748fcf9-9ab4-4ea1-a4b2-6f48fe1aaf67)